tests/testthat/test-tidy-upkeep.R

```diff
@@ -1,4 +1,16 @@
 test_that("upkeep bullets don't change accidentally", {
+  withr::local_options(
+    usethis.description = list(
+      "Authors@R" = utils::person(
+        "Jane", "Doe",
+        email = "jane@rstudio.com",
+        role = c("aut", "cre")
+      ),
+      License = "MIT + file LICENSE"
+    )
+  )
+  create_local_package()
```

Comment:
I encountered this failure too, thanks!

Reply:
I fixed it [here](https://github.com/r-lib/usethis/pull/1798/commits/7f16ed2842912a8fe496b4fbf3276706141159bd) as well but that's not ready anyway...